docs/pages/blog/turbo-1-6-0.mdx

```diff
@@ -0,0 +1,209 @@
+
+So we're excited to announce that **any codebase can now use Turborepo**. Try it out now:
+
+### Add Turborepo to your project
+
+1. **Install `turbo` globally:**
+
+<Tabs items={['npm', 'yarn', 'pnpm']} storageKey="selected-pkg-manager">
+  <Tab>
+    ```bash
+    npm install --global turbo
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    yarn global add turbo
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    pnpm install --global turbo
+    ```
+  </Tab>
+</Tabs>
+
+2. **Add a `turbo.json` file at the base of your new repository:**
+
+<Tabs items={['Next.js', 'Vite']} storageKey="selected-framework">
+  <Tab>
+```json filename="turbo.json"
+{
+  "pipeline": {
+    "build": {
+      "outputs": [".next/**"]
+    },
+    "lint": {
+      "outputs": []
+    }
+  }
+}
+```
+  </Tab>
+  <Tab>
+```json filename="turbo.json"
+{
+  "pipeline": {
+    "build": {
+      "outputs": ["dist/**"]
+    },
+    "lint": {
+      "outputs": []
+    }
+  }
+}
+```
+
+Some Vite starters ship with a `package.json` that looks like this:
+
+```json filename="package.json"
+{
+  "scripts": {
+    "build": "tsc && vite build"
+  }
+}
+```
+
+We recommend splitting these into a `lint` and `build` script.
+
+```json filename="package.json"
+{
+  "scripts": {
+    "build": "vite build",
+    "lint": "tsc"
+  }
+}
+```
+
+This means that Turbo can schedule them separately.
+
+  </Tab>
+</Tabs>
+
+3. **Try running `build` and `lint` with `turbo`:**
+
+```bash
+turbo build lint
+```
+
+Congratulations - **you just ran your first build with `turbo`**. You can try:
+
+- Running through the full [Quickstart](/docs/getting-started/add-to-project).
+- Check out our updated [Core Concepts docs](/docs/core-concepts/caching) to understand what makes Turborepo special.
+
+## When should I use Turborepo?
+
+Turborepo being available for non-monorepos opens up a lot of new use cases. But when is it at its best?
+
+### When scripts depend on each other
+
+You should use Turbo to run your `package.json` scripts. If you've got multiple scripts which all rely on each other, you can express them as Turbo tasks:
+
+```json filename="turbo.json"
+{
+  "pipeline": {
+    "build": {
+      "outputs": ["dist/**"]
+    },
+    "lint": {
+      // 'build' should be run before 'lint'
+      "dependsOn": ["build"]
+    },
+    "test": {
+      // 'build' should be run before 'test'
+      "dependsOn": ["build"]
+    }
+  }
+}
+```
+
+Then, you can run:
+
+```sh
+turbo run lint test
+```
+
+Because you've said that `build` should be run before `lint` and `test`, it'll _automatically run `build` for you_ when you run `lint` or `test`.
+
+Not only that, but it'll figure out the optimal schedule for you. Head to our core concepts doc on [optimizing for speed](/docs/core-concepts/monorepos/running-tasks#most-tools-dont-optimize-for-speed).
+
+### When you want to run tasks in parallel
```

Comment:
This is actually a really clever little layout showing what's possible even without our upcoming tweaks.